wlm/src/test/java/ai/djl/serving/wlm/ModelInfoTest.java

```diff
@@ -266,4 +268,122 @@ public void testInferLMIEngine() throws IOException, ModelException {
         model = new ModelInfo<>("build/models/lmi_test_model");
         Assert.assertThrows(model::initialize);
     }
+
+    @Test
+    public void testInferSageMakerEngine() throws IOException, ModelException {
+        Path modelStore = Paths.get("build/models");
+        Path modelDir = modelStore.resolve("sagemaker_test_model");
+        Files.createDirectories(modelDir);
+
+        // test case for valid xgboost model with no customized schema and inference spec
+        Map<String, String> validXGBoostMetaDataMap =
```

Comment:
This is only four small yaml files in this test. You should save those into real files in `test/resources` rather than writing them as part of the test. Our concerns with test files are just that big things like images or model parameters will increase the git clone time. Anything of that size is perfectly fine